src/js/storyline.js

```diff
@@ -130,6 +152,27 @@ Storyline.prototype = {
   },
   positionSlider: function(slider) {
     this.elem.appendChild(slider.elem);
+  },
+  attachClickHandler: function(div, targetType) {
+    storyline.trackEvent('click', targetType)
+    var self = this;
+    for(var i=0; i < div.length; i++) {
+      div[i].onclick = function(event) {
+        self.handleClick(event, targetType);
+      }
+    }
+  },
+  handleClick: function(event, targetType) {
+    storyline.trackEvent('click', targetType)
```

Comment:
we should eliminate all references to a global `storyline` in favor of `this` (or `this.storyline` for components)